javagen/src/main/java/com/azure/autorest/model/javamodel/JavaPackage.java

```diff
@@ -213,6 +218,20 @@ public final void addPom(String name, Pom pom) {
         xmlFiles.add(xmlFile);
     }
 
+    public final void addJavaFromResources(String packageName, String name) {
```

Comment:
Add a convenience method to load Java file resources and write out, since Alan got such use case as well.

Reply:
Why do we need this helper class to be loaded from resources?

Reply:
It is simipler to maintain the helper class this way, when the whole class is to be outputted as generated code.

E.g. [MultipartFormDataHelper](https://github.com/Azure/autorest.java/blob/b49056c5e03488dd61ee8884597014179df10a3a/javagen/src/main/resources/MultipartFormDataHelper.java) from codegen resources is outputted as [code](https://github.com/Azure/autorest.java/blob/b49056c5e03488dd61ee8884597014179df10a3a/typespec-tests/src/main/java/com/cadl/multipart/implementation/MultipartFormDataHelper.java) in test case (the only diff is package name and header).